aframe/src/index.js

```diff
@@ -8,8 +8,9 @@ AFRAME.registerComponent('zesty-ad', {
   schema: {
     adSpace: { type: 'string' },
     creator: { type: 'string' },
-    adURI: { type: 'string' },
-    url: { type: 'string' },
+    defaultAd: { type: 'string', default: 'square' },
```

Comment:
you can add:
`oneOf: ['tall', 'wide', 'square']`

Although ideally, all the format declarations (arrays, objects, etc.) would import from a file like `utils/formats.js`.

https://github.com/ngokevin/aframe-core/blob/master/docs/components.md#component-schema